cmd/create/cluster/cmd.go

```diff
@@ -551,7 +551,11 @@ func run(cmd *cobra.Command, _ []string) {
 	r := rosa.NewRuntime().WithOCM()
 	defer r.Cleanup()
 
-	awsClient := aws.GetAWSClientForUserRegion(r.Reporter, r.Logger)
+	supportedRegions, err := r.OCMClient.GetDatabaseRegionList()
```

Comment:
NP
I wonder if we should encapsulate this into `GetAWSClientForUserRegion`, WDYT?

Reply:
Cyclic import if not mistaken

Reply:
That would be cyclic import, so we would need to refactor some more